$NetBSD: patch-aa,v 1.2 2005/12/03 04:18:59 dmcmahill Exp $

A double semicolon counts as an empty statement, after which ISO C90
does not allow any further variable declarations.

--- src/gui-netlist-window.c.orig	Tue Mar  1 06:33:01 2005
+++ src/gui-netlist-window.c	Thu Aug  4 18:06:36 2005
@@ -692,7 +692,7 @@ gui_netlist_highlight_node(gchar *node_n
 	{
 	GtkTreePath			*path;
 	GtkTreeIter			iter;
-	LibraryMenuType		*net;;
+	LibraryMenuType		*net;
 	gchar				*name;
 
 	if (!node_name)
